apps/web/lib/app-providers-app-dir.tsx

```diff
@@ -276,7 +279,16 @@ const AppProviders = (props: PageWrapperProps) => {
                   <OrgBrandProvider>
                     {/* @ts-expect-error FIXME remove this comment when upgrading typescript to v5 */}
                     <CacheProvider>
-                      <MetaProvider>{props.children}</MetaProvider>
+                      <MetaProvider>
+                        {(isENVDev || IS_CALCOM) && (
+                          <DubAnalytics
+                            cookieOptions={{
+                              domain: isENVDev ? undefined : new URL(WEBAPP_URL).hostname,
```

Comment:
I think this check should also check for IS_CALCOM too right?

Reply:
I believe we are checking for `IS_CALCOM` in the conditional above, or am I misunderstanding something here? 👀

Reply:
I think i miss understood what the domain option here is doing - makes sense now. Sorry :D